ledger/spverificationtracker.go

```diff
@@ -134,59 +124,66 @@ func (spt *stateProofVerificationTracker) prepareCommit(dcc *deferredCommitConte
 	return nil
 }
 
-func (spt *stateProofVerificationTracker) commitRound(ctx context.Context, tx *sql.Tx, dcc *deferredCommitContext) (err error) {
-	err = insertStateProofVerificationContext(ctx, tx, dcc.spVerification.CommitContext)
-	if err != nil {
-		return err
+func (spt *spVerificationTracker) commitRound(ctx context.Context, tx *sql.Tx, dcc *deferredCommitContext) (err error) {
+	if len(dcc.spVerification.CommitContext) != 0 {
+		err = commitSPContexts(ctx, tx, dcc.spVerification.CommitContext)
+		if err != nil {
+			return err
+		}
 	}
 
 	if dcc.spVerification.LastDeleteIndex >= 0 {
-		err = store.DeleteOldStateProofVerificationContext(ctx, tx, dcc.spVerification.EarliestLastAttestedRound)
+		err = store.CreateSPVerificationAccessor(tx).DeleteOldSPContexts(ctx, dcc.spVerification.EarliestLastAttestedRound)
```

Comment:
CreateSPVerificationAccessor -> MakeSPVerificationAccessor

Ignacio has renamed or going to rename rest of Create* to Make*